Automattic/newspack-plugin
# Newspack Logo Above Wizards

### All Submissions:






### Changes proposed in this Pull Request:

This PR places a Newspack logo above each Wizard, and cleans up some inconsistencies between different Wizard styles. 

Closes #146 .

### How to test the changes in this Pull Request:

1. `npm ci && npm run clean && npm run build:webpack`
2. Click through every possible Wizard and Checklist and confirm that all have the correct logo at the top and have equivalent spacing. Note that the Set Up Wizard has the full logo while all others have only the blue circle.

### Other information:
